CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## Upcoming
+## v0.7.0 (Upcoming)
```

Comment:
I was thinking to resolve these two issues (#29 and #18) to make a real `1.0.0` version